bing-dict.el

```diff
@@ -133,8 +133,52 @@ The value could be `synonym', `antonym', `both', or nil.")
                                          'face
                                          'font-lock-builtin-face))
 
+(defvar bing-dict--org-file "~/.orgs/vocabulary.org"
+  "The file where store the vocabulary.")
+
+(defvar bing-dict--org-file-title "Vocabulary"
+  "The title of the vocabulary org file.")
+
+(defvar bing-dict--save-search-result nil
+  "save bing dict search result or not.")
+
+
+(defun bing-dict--tidy-headlines ()
+  "remove extra spaces between stars and the headline text"
+  (save-excursion
+    (goto-char (point-min))
+    (while (re-search-forward "^\\*+\\([[:space:]][[:space:]]+\\)" (point-max) t)
+      (replace-match " " nil nil nil 1))))
+
+(defun bing-dict--save-word (word definition)
+  "save word in org file"
+  (find-file bing-dict--org-file)
```

Comment:
I'd like to, but it seems there is no better way to insert an org heading without using `org-insert-heading` or `org-insert-subheading`?